lib/admin/app.rb

```diff
@@ -23,6 +23,10 @@ class App < Sinatra::Base
       end
     end
 
+    get '' do
```

Comment:
But then when we use `/admin` (without the trailing slash) the link for the 'start now' button points to `/actions` when it should point to `/admin/actions` - oh the joy of Sinatra